packages/utils/src/translation-helpers.ts

```diff
@@ -165,11 +166,10 @@ export function getTranslateableFields(
         const localizedTextInputs = el.meta.localizedTextInputs as string[];
         if (localizedTextInputs && Array.isArray(localizedTextInputs)) {
           localizedTextInputs
-            .filter(input => el.component?.options?.[input]?.['@type'] === localizedType)
+            .filter(input => get(el.component?.options || {}, `${input}.@type`) === localizedType)
```

Comment:
How is this different from the previous code?

Reply:
lodash `get` accepts path to fetch an item in an object - could be "title.text". 
Not possible via direct `[]`